src/business.ts

```diff
@@ -80,38 +95,51 @@ export class BusinessApi {
     error?: MarqetaError,
   }> {
     const searchOptions = snakecaseKeys(search)
-    const resp = await this.client.fire(
-      'GET',
+    const resp = await this.client.fire('GET',
       'businesses',
       { ...searchOptions }
     )
-    // catch 404s...
-    if (resp?.payload?.errorCode >='404000' && resp?.payload?.errorCode <= 404999) {
-      const error = resp?.payload?.errorMessage
+    // catch any errors...
+    if (resp?.payload?.errorCode) {
       return {
         success: false,
         error: {
           type: 'marqeta',
-          error,
+          error: resp?.payload?.errorMessage,
           status: resp?.payload?.errorCode,
-          marqetaStatus: resp?.payload?.errorCode
         },
       }
     }
-    // catch all other errors...
-    if (resp?.payload?.errorCode > 404999){
-      const error = resp?.payload?.errorMessage
+    return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
+  }
+
+  /*
+   * Function to take the attributes of a new Business account, create that
+   * in Marqeta, and return the account information. If no attributes are
+   * provided, a new Business account will still be created and its
+   * token ID returned.
+   */
+  async create(business: Business): Promise<{
+    success: boolean,
+    body?: Business,
+    error?: MarqetaError,
+  }> {
```

Comment:
One suggestion to consider:

```typescript
   async create(business: Partial<Business>): Promise<{
     success: boolean,
     body?: Business,
     error?: MarqetaError,
   }> {
```

Where you then don't have to worry about the required fields in the `Business` interface... but it's just an idea. This is fine.

Reply:
added